tests/test_minimal/test_tools/test_expand_paths.py

```diff
@@ -104,6 +104,150 @@ def test_expand_paths(tmpdir):
     assert len(string_directory_out) == len(expected)
 
 
+def test_expand_paths_with_extras(tmpdir='tmp'):
```

Comment:
We use pytest in our testing suite and we have it in the requirements as far as I remember. Would it not be more robust to use their dedicated fixtures instead of an argument?

https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
Ah, thank you! I didn't know about this feature. I go rid of the arguments.